### PR TITLE
feat/axeptio: onboard web device mode destination axeptio

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## PR Description

Onboard new web device mode destination Axeptio

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/743)

### CODE REVIEW

File: integrations/Axeptio/browser.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
```
Comment (ItsSudip) [inline integrations/Axeptio/browser.js:4]: We should update the new `logger` here. You can take reference from amplitude.

File: integrations/Axeptio/browser.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded Axeptio===");
+    return !!window.__axeptioSDK && typeof window.__axeptioSDK === "object";
+  }
+
+  isReady() {
+    logger.debug("===In isReady Axeptio===");
+    this.recordAxeptioEvents();
```
Comment (ItsSudip) [inline integrations/Axeptio/browser.js:40]: We should provide an option in the dashboard to check if the customer wants to use the callback functionality.

File: integrations/Axeptio/browser.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    return !!window.__axeptioSDK;
+  }
+
+  // this fucntion is used to record the triggered axeptio events through callback
+  recordAxeptioEvents() {
```
Comment (ItsSudip) [inline integrations/Axeptio/browser.js:45]: Are we not filtering out the events? I mean we are setting callbacks for all the events by default?

**Review by ItsSudip — CHANGES_REQUESTED** [review]

File: integrations/Axeptio/browser.js (hunk 2 above)
Comment (ujjwal-ab) [reply integrations/Axeptio/browser.js:40]: Yeah, we can do that. But, as the callback is the only supported feature here. Will it make sense?

File: integrations/Axeptio/browser.js (hunk 3 above)
Comment (ujjwal-ab) [reply integrations/Axeptio/browser.js:45]: Yeah, as we are not totally sure about the event's name. We haven't added any filters and the callback is set to all the triggered events. One check is there for the payload. If the payload is undefined we are only sending the event name else both.

File: integrations/Axeptio/browser.js (hunk 2 above)
Comment (ItsSudip) [reply integrations/Axeptio/browser.js:40]: Yes. Someone may only want to load the axeptio SDK.

File: integrations/Axeptio/browser.js (hunk 3 above)
Comment (ItsSudip) [reply integrations/Axeptio/browser.js:45]: Okay let's support all the events later we can add some filters as an enhancement.

File: integrations/Axeptio/browser.js (hunk 1 above)
Comment (ujjwal-ab) [reply integrations/Axeptio/browser.js]: Updated

File: integrations/Axeptio/browser.js (hunk 2 above)
Comment (ujjwal-ab) [reply integrations/Axeptio/browser.js]: Updated

**Review by ItsSudip — APPROVED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Review by aashishmalik — APPROVED** [review]
LGTM

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**